requirements.txt

```diff
@@ -0,0 +1,72 @@
+alabaster==0.7.12
+attrs==19.3.0
+Babel==2.7.0
+bleach==3.1.0
+bokeh==1.4.0
+certifi==2019.11.28
+chardet==3.0.4
+codecov==2.0.15
+coverage==4.5.4
+cycler==0.10.0
+decorator==4.4.1
+defusedxml==0.6.0
+docutils==0.15.2
+entrypoints==0.3
+idna==2.8
+imagesize==1.1.0
+importlib-metadata==1.3.0
+ipython-genutils==0.2.0
+Jinja2==2.10.3
+jsonschema==3.2.0
+jupyter-core==4.6.1
+kiwisolver==1.1.0
+latexcodec==1.0.7
+MarkupSafe==1.1.1
+matplotlib==3.1.2
+mistune==0.8.4
+more-itertools==8.0.2
+nbconvert==5.6.1
+nbformat==4.4.0
+nbsphinx==0.5.0
+numpy==1.17.4
+numpydoc==0.9.1
+oset==0.1.3
+packaging==19.2
+pandas==0.25.3
+pandocfilters==1.4.2
+Pillow==6.2.1
+pkg-resources==0.0.0
+pluggy==0.13.1
+py==1.8.0
+pybtex==0.22.2
+pybtex-docutils==0.2.2
+Pygments==2.5.2
+pyparsing==2.4.5
+pyrsistent==0.15.6
+pytest==5.3.1
+pytest-cov==2.8.1
+python-dateutil==2.8.1
+pytz==2019.3
+PyYAML==5.2
+requests==2.22.0
+scipy==1.3.3
+six==1.13.0
+snowballstemmer==2.0.0
+Sphinx==2.2.2
+sphinx-bootstrap-theme==0.7.1
+sphinxcontrib-applehelp==1.0.1
+sphinxcontrib-bibtex==1.0.0
+sphinxcontrib-devhelp==1.0.1
+sphinxcontrib-htmlhelp==1.0.2
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.2
+sphinxcontrib-serializinghtml==1.1.3
+testpath==0.4.4
+toml==0.10.0
+tornado==6.0.3
+traitlets==4.3.3
+urllib3==1.25.7
+wcwidth==0.1.7
+webencodings==0.5.1
+xlrd==1.2.0
+zipp==0.6.0
```

Comment:
Hi @rodrigomoliveira1 , this list was generated automatically via a VS Code extension, and it identified the currently installed software versions. Most of them are already installed by default by the "big" ones, like numpy, scipy and matplotlib, for example. We made this simply to facilitate the installation of the extensions needed in the multiple platforms and IDEs currently in use in our lab.

Reply:
I see. But if they are default by the big ones, there's no need to add on ross' requirements.
Once you install ross, it will search for numpy (for e.g) and while installing numpy, it will search for its dependancies. 
Did you experience any problem, related to python packages, while installing ross and using it on multiple IDEs?

Reply:
Unfortunately yes, and almost exclusively on windows machines. The developers on the lab are using linux, and it works fine there now, but on the beginning we had some troubles with windows.

Reply:
I believe this file is not needed, but we left it there "just in case".

Reply:
> Unfortunately yes, and almost exclusively on windows machines. The developers on the lab are using linux, and it works fine there now, but on the beginning we had some troubles with windows.

We can open an issue reporting this problem, and then we invastigate what's going wrong on windows machines